src/source/feeds.rst

```diff
@@ -13,16 +13,67 @@ We treat the provided XML file as the desired list of ads the user wants to have
+:ref:`feed_categoryId`                 category identifier                  numeric, positive    yes       
+:ref:`feed_status`                     desired status (default ACTIVE)      ACTIVE,PAUSED        no       
+:ref:`feed_url`                        product URL                          max. 2048 chars      no        
+:ref:`feed_vanityUrl`                  displayed URL                        max. 256 chars       no        
+:ref:`feed_priceType`                  sales model for product              enum                 yes       
+:ref:`feed_price`                      product price in cents if applicable positive integer     yes/no       
+:ref:`feed_originalPrice`              original price before discount       positive integer     no        
+:ref:`feed_media`                      product images                       :ref:`feed_media`    no
+:ref:`feed_attr`                       collection of ad attributes          :ref:`feed_attr`     no        
+:ref:`feed_budget`                     budget details                       :ref:`feed_budget`   no        
+:ref:`feed_ship`                       shipping options                     :ref:`feed_ship`     no
+:ref:`feed_phoneNumber`                phone number                         max. 32 chars        no        
+:ref:`feed_emailAdvertiser`            allow emails to the seller           true,false           no
+:ref:`feed_regionId`                   only applicable for Kijiji Canada    numeric              no        
+:ref:`feed_engines`                    targeted search engines for this ad  emunerated list      no
+:ref:`feed_microTip`                   tiny product hightlight              max. 18 chars        no
+:ref:`feed_mpn`                        Manufacturer Part Number (MPN)       len. 2-70 chars      no   
+:ref:`feed_googleProductCategory`      google category for your product     string               no
+:ref:`feed_productType`                customer product type                max. 750 chars       no    
+:ref:`feed_brand`                      product brand name                   max. 70 chars        no
+:ref:`feed_gtin`                       Global Trade Identification Number   max. 50 chars        no  
+:ref:`feed_itemGroupId`                groups product variants in your      len. 1-50 chars      no
+:ref:`feed_condition`                  condition of product                 enum                 no
+:ref:`feed_material`                   main product fabric or material      max. 200 chars       no
+:ref:`feed_energyEfficiencyClass`      energy efficiency class              enum                 no
+:ref:`feed_minEnergyEfficiencyClass`   minimal energy efficiency class      enum                 no
+:ref:`feed_maxEnergyEfficiencyClass`   maximal energy efficiency class      enum                 no
+:ref:`feed_color`                      product color                        len. 1-100 chars     no
```

Comment:
keep consistency: `max` vs `len` ? especially since they are all optional. If the GTIN is acceptable as empty, why would the color not be?

Reply:
that is how it was described in merchant shop, sometimes they state minimal 1. I read it: can't be empty if provided. For MPN it probably makes sense to require more than 2 characters, for the others... I will remove that information from doc

Reply:
👍